site/components/pages/Pledge/PledgeDashboard/Cards/RetirementsCard/RetirementsChart.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { FC } from "react";
+import {
+  Bar,
+  BarChart,
+  LabelList,
+  ResponsiveContainer,
+  Tooltip,
+  TooltipProps,
+  XAxis,
+  YAxis,
+} from "recharts";
```

Comment:
my knowledge of webpack is weak, but I assume there is no tree-shaking here meaning the entire recharts library is bundled. 

does recharts offer tree-shaking compatible imports like some libraries do?
`import Bar from "recharts/Bar"`